src/dataframerow/utils.jl

```diff
@@ -236,30 +236,20 @@ function row_group_slots(cols::NTuple{N,<:Union{CategoricalVector,PooledVector}}
     return ngroups, UInt[], Int[], sorted
 end
 
-# Builds RowGroupDict for a given DataFrame.
```

Comment:
A general question: does whole this code belong to `DataFrameRow`? It seems not - maybe we should move it to other part of the codebase?

Reply:
Yes, it's here for historical reasons I think. I always feel it's not the right place. Let's fix this as part of https://github.com/JuliaData/DataFrames.jl/pull/2050.